<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutHotel</class>
 <widget class="QDialog" name="AjoutHotel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDateEdit {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QLineEdit
{

    border: 1px solid gray;
    border-radius: 4px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QDialog#AjoutHotel
{background-image: url(:/projetimages/projetimages/a.jpg);

}
*{
font-family:centry gothlic ;
font-size:15px;
}
QWidget#widget
{

	background:rgba(0,0,0,0.7);
border-radius:15px;
}




#label{
color:white;
background:transparent;

}

#label_2
{
color:white;
background:transparent;

}


#label_3{
color:white;
background:transparent;

}

#label_4{
color:white;
background:transparent;

}

#label_5{
color:white;
background:transparent;

}

#label_6{
color:white;
background:transparent;

}

#label_7{
color:white;
background:transparent;

}

#label_8{
color:white;
background:transparent;

}
#label_9{
color:white;
background:transparent;

}#label_10{
color:white;
background:transparent;

}#label_11{
color:white;
background:transparent;

}
QPushButton
 {
     background-color: red;

     border-radius: 10px;

 }

QPushButton
{
color:white;

border-raduis:15px;
}
QPushButton:hover{

color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>941</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="prenomadmin_txt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomhotel_txt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmer le mot de passe :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Catégorie :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomadmin_txt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cinadmin_txt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="categoriehotel_txt">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de passe :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CIN de l'administrateur :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mdpadmin1_txt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'administrateur :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginadmin_txt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="datenaissance_txt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse de l'administrateur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prénom de l'administrateur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date de naissance :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mdpadmin2_txt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="adressehotel_txt">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'hôtel :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="adresseadmin_txt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse de l'hotel :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Login :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>401</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/projetimages/projetimages/e.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterhotel2_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>560</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
